rkt/rkt.go

```diff
@@ -189,11 +198,43 @@ func getTabOutWithWriter(writer io.Writer) *tabwriter.Writer {
 	return aTabOut
 }
 
+func startProfile() (cpufile *os.File, memfile *os.File, err error) {
+	if globalFlags.CPUProfile != "" {
+		cpufile, err = os.Create(globalFlags.CPUProfile)
+		if err != nil {
+			return nil, nil, fmt.Errorf("cannot create cpu profile file %q: %v", globalFlags.CPUProfile, err)
+		}
+		pprof.StartCPUProfile(cpufile)
+	}
+	if globalFlags.MemProfile != "" {
+		memfile, err = os.Create(globalFlags.MemProfile)
+		if err != nil {
+			return nil, nil, fmt.Errorf("cannot create memory profile file %q: %v", globalFlags.MemProfile, err)
+		}
+	}
+	return cpufile, memfile, nil
+}
+
+func stopProfile(cpuprofile, memprofile *os.File) {
+	pprof.StopCPUProfile()
+	pprof.WriteHeapProfile(memprofile)
+	cpuprofile.Close()
+	memprofile.Close()
+}
+
 // runWrapper returns a func(cmd *cobra.Command, args []string) that internally
 // will add command function return code and the reinsertion of the "--" flag
 // terminator.
 func runWrapper(cf func(cmd *cobra.Command, args []string) (exit int)) func(cmd *cobra.Command, args []string) {
 	return func(cmd *cobra.Command, args []string) {
+		cpufile, memfile, err := startProfile()
+		if err != nil {
+			stderr.Printf("cannot setup profiling %v", err)
```

Comment:
Could this use `stderr.PrintE()` to be consistent with the rest? Or is it too early and `--debug` isn't parsed yet?

If `stderr.PrintE()` cannot be used, there should be a `:` before the `%v`.